actiontext/lib/tasks/actiontext.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+namespace :action_text do
+  desc "Copy over the migration, stylesheet, and JavaScript files"
+  task install: %w( environment run_action_text_generator)
+
+  task :run_action_text_generator do
+    system "#{RbConfig.ruby} ./bin/rails generate action_text:install"
```

Comment:
Hm, this doesn't seem right to me. Though I can see we're doing the same thing in the Action Mailbox install Rakefile.

Can we require the correct generator and call `start` on it like we do in the credentials command? (Assuming that'll still trigger the test framework hook.)

@y-yagi would you happen to have any opinions here?

Reply:
I tried requiring the generator and running it. The test framework hook didn't get invoked.

Reply:
I agree that we shouldn't run it from the bin file. Would [this method](https://github.com/rails/rails/blob/64ff6bc4c657ae1a1f46e108ad76cf7825a07f55/railties/lib/rails/generators/actions.rb#L224) be of help in this case?

Reply:
That method is only to be run within a generator, not from a Rake task.

Reply:
I see. How about [this one](https://github.com/rails/rails/blob/3578f6929bd3065ff46622a440f055a566f4518a/railties/lib/rails/commands/generate/generate_command.rb#L26)? Could that be invoked from a rake task?

Reply:
Then I'd prefer to just do something like:

```ruby
task "action_text:install" do
  require "rails/command" # Can't remember the require path or if Rake tasks already have access to Rails::Command.
  Rails::Command.invoke "generate action_text:install"
end
```

Reply:
Did the necessary changes. Removing `desc` hide a generator from the list.